app/views/downloads/_download_afterword.html.erb

```diff
@@ -23,5 +23,5 @@
     </div>
   <% end %>
 
-  <p class="message"><%= ts("Please") %> <%= link_to ts("drop by the archive and comment"), new_work_comment_url(@work) %> <%= ts("to let the author know if you enjoyed their work!") %></p>
+  <p class="message"><%= t(".please_comment_html", work_comment_link: link_to(t(".work_comment"), new_work_comment_url(@work))) %></p>
```

Comment:
So @brianjaustin and I were talking about i18n and download caching, and we think it might be a good idea to explicitly pass the locale for now. Could you add `locale: :en` here, [like we do with zh-CN in the proxy notice](https://github.com/otwcode/otwarchive/blob/801ce0931c0dbb144f9631eed5e9cf743a6c8d00/app/views/layouts/_proxy_notice.html.erb#L9)?